<?xml version="1.0" ?>
<sdf version="1.5">
    <model name="dummy_arch">
        <static>1</static>
        <pose>0.0 0.0 0.1 0.0 0.0 0.0</pose>
        <link name="link">
                <kinematic>0</kinematic>
                <inertial>
                        <mass>1</mass>
                        <inertia>
                                <ixz>0</ixz>
                                <ixx>10.0</ixx>
                                <ixy>0</ixy>
                                <izz>10.0</izz>
                                <iyy>1</iyy>
                                <iyz>10.0</iyz>
                        </inertia>
                </inertial>
                <self_collide>0</self_collide>
                <gravity>1</gravity>
                <collision name="collision">
                        <geometry>
                            <mesh>
                                <uri>model://dummy_arch/dummy_arch.stl</uri>
                            </mesh>
                        </geometry>
                        <max_contacts>10</max_contacts>
                        <surface>
                                <contact>
                                        <ode></ode>
                                </contact>
                                <bounce></bounce>
                                <friction>
                                        <ode></ode>
                                </friction>
                        </surface>
                </collision>
                <visual name="visual">
                        <geometry>
                            <mesh>
                                <uri>model://dummy_arch/dummy_arch.stl</uri>
                            </mesh>
                        </geometry>
                        <material>
                                <script>
                                        <uri>file://media/materials/scripts/gazebo.material</uri>
                                        <name>Gazebo/Grey</name>
                                </script>
                        </material>
                </visual>
                <velocity_decay>
                        <linear>0</linear>
                        <angular>0</angular>
                </velocity_decay>
        </link>
    </model>
</sdf>

